main.css

```diff
@@ -1,393 +0,0 @@
-.pageContainer {
-      display: grid;
-      grid-template-columns: 20fr 20fr 20fr 20fr 20fr;
-      grid-gap: 1rem;
-      background-color: #ffffe6;
-      border-color: #002633;
-      border: .5em;
-      color: #0d0d0d;
-}
-
-/**universal attributes***/
-
-p {
-      justify-content: space-around;
-      font-family: 'Libre Baskerville', 16px;
-      text-align: left;
-}
-
-h2 {
-      display: flex;
-      text-align: center;
-}
-
-h3 {
-      display: flex;
-}
-
-.linkButton {
-      display: flex;
-      justify-content: center;
-      align-content: center;
-      margin-bottom: 1em;
-      margin-top: 2.5em;
-}
-
-a:hover {
-      color: hotpink;
-}
-
-a {
-      text-decoration: none;
-}
-
-/********Header**********/
-
-.headerContainer {
-      display: grid;
-      grid-column-start: 2;
-      grid-column-end: 5;
-      order: 1;
-}
-
-#header {
-      display: flex;
-      flex-direction: column;
-      border-style: none none double none;
-      border-color: #002633;
-}
-
-#header-title {
-      display: flex;
-      font-size: 4em;
-      color: #0d0d0d;
-      justify-content: center;
-      text-decoration: none;
-      transition: 2s;
-}
-
-#header-description {
-      display: flex;
-      font-size: 1.25em;
-      justify-content: center;
-}
-
-/***********Picture gallery *******/
-
-.pictureContainer {
-      display: grid;
-      grid-column-start: 2;
-      grid-column-end: 5;
-      order: 2;
-}
-
-.teaserBox {
-      border-style: none none double none;
-      border-width: .5em;
-}
-
-.pictureGallery {
-      display: flex;
-      flex-direction: column;
-      margin-top: 1em;
-}
-
-#pictures-title {
-      display: flex;
-      font-size: 1.5em;
-      justify-content: center;
-      align-content: center;
-}
-
-.mainPageGallery {
-      display: flex;
-      flex-direction: row;
-      justify-content: space-around;
-      align-content: center;
-}
-
-.picturesLabels {
-      display: flex;
-      flex-direction: row;
-      justify-content: space-around;
-      font-size: 1em;
-}
-
-.picture {
-      display: flex;
-}
-
-/*********** quote section *******/
```

Comment:
Good comments throughout your CSS, it makes it very easy to find styles for different parts of your pages